pkg/controller/dataimportcron-controller.go

```diff
@@ -259,16 +259,22 @@ func splitImageStreamName(imageStreamName string) (string, string, error) {
 	return "", "", errors.Errorf("Illegal ImageStream name %s", imageStreamName)
 }
 
-func (r *DataImportCronReconciler) pollImageStreamDigest(ctx context.Context, dataImportCron *cdiv1.DataImportCron) (reconcile.Result, error) {
+func (r *DataImportCronReconciler) pollDigest(ctx context.Context, dataImportCron *cdiv1.DataImportCron) (reconcile.Result, error) {
```

Comment:
Why renamed? it is for image streams and the diff is commented out.

Reply:
I tried to implement it differently to the HTTP import as we initially wanted with the timestamp of the last successful if-modified-since request, but then I changed it to digest as the URL suffix as suggested in the Jira card so I moved those changes to a comment.
Don't know if to keep them